packages/core/src/controllers/subscriber.ts

```diff
@@ -144,29 +135,22 @@ export class Subscriber extends ISubscriber {
 
   private onEnable() {
     this.cached = [];
-    this.events.emit(SUBSCRIBER_EVENTS.enabled);
```

Comment:
Are we sure this isn't needed? Are other controllers relying on this being emitted?

cc @pedrouid

Reply:
Not needed, this was internal event signifying that initialization completed. All of these were replaced with a boolean flag instead and made consistent to not rely on events like this in all controllers